pcsx2/SPU2/Windows/SPU2.rc

```diff
@@ -186,6 +181,17 @@ BEGIN
 END
 #endif    // APSTUDIO_INVOKED
 
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// AFX_DIALOG_LAYOUT
+//
+
+IDD_CONFIG AFX_DIALOG_LAYOUT
+BEGIN
+    0
+END
+
```

Comment:
Where'd this come from?

Reply:
If you use the vs designer it will auto-create lines like that.

Reply:
I figured that, thanks Red. It needs to be removed

Reply:
Thanks, I took care of it.